### ISSUE TITLE
I cannot install this very useful extension because it isn't signed by the Extension Marketplace

### ISSUE THREAD

**Issue by PetrKryslUCSD** [stkb/Rewrap#413]
Could this be addressed? Many thanks!

**Comment by dalps** [comment]
The install option is available from the "Manage" menu - the little cog.

![image](https://github.com/user-attachments/assets/a5646dad-474a-45e1-b24d-ef1bb51f9dd7)

**Comment by PetrKryslUCSD** [comment]
Yes, it is. But, I have an automatic procedure https://github.com/PetrKryslUCSD/VSCode_Julia_portable/blob/859b4fa413c96e88d28d4a5d56c6344e1ddf543a/assets/install.sh#L159
It fails there...

**Comment by haberdashPI** [comment]
Though this is a reasonable workaround (for most use cases), what is the rationale for not having this be signed by the Extension Marketplace?

**Comment by lifeisafractal** [comment]
@dalps thanks for the work around (that got me going). Judging by the last commit in the repo (2 years ago), I'd say the author isn't maintaining the project at the moment. The author also hasn't had any activity on Github in almost that much time also. Check out #392

**Comment by karlhorky** [comment]
@PetrKryslUCSD Now that @dnut forked Rewrap ([announcement comment](https://github.com/stkb/Rewrap/issues/392#issuecomment-2552234144)), probably good to report this issue over in the fork too:

https://github.com/dnut/Rewrap

@dnut maybe you can enable Issues in the fork.

This feature may be another thing to prioritize for a good baseline starting point for the fork.

**Comment by dnut** [comment]
> @dnut maybe you can enable Issues in the fork.

done

> This feature may be another thing to prioritize for a good baseline starting point for the fork.

Publishing the extension is my first goal, which I believe will address this issue.

**Comment by PetrKryslUCSD** [comment]
Fantastic!

**Comment by dnut** [comment]
I published the extension as "Rewrap Revived" with the same release and pre-release versions.

https://marketplace.visualstudio.com/items?itemName=dnut.rewrap-revived

I suggest installing the pre-release version so you can identify any bugs and report them so they don't make their way into the stable release. If you report any observed bugs, you can switch to the stable release and rest assured that the bug won't appear there.
